docs/application/native/guides/app-management/tidl.md

```diff
@@ -16,12 +16,18 @@ Usage:
   -h, --help                  Show help options.
   -l, --language=LANGUAGE     Select generating language (C, C++, C#).
   -i, --input=INPUT           A tidl interface file.
-  -o, --output=OUTPUT         The generated interface file.
+  -o, --output=OUTPUT         Generate an interface file.
   -p, --proxy                 Generate proxy code.
   -s, --stub                  Generate stub code.
+  -r, --rpclib                Generate C# library for rpc-port.
+  -t, --thread                Generate thread code with stub code.
+  -b, --beta                  Use beta version for private file sharing.
   -v, --version               Show version information.
 
 ```
+  > **Note**
+  >
+  > Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.
```

Comment:
Thanks for changing format to note format.

However, the note format for Tizen document was changed. You can refer [here](https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md).

Please change the note style as like the previous comment.

\> \[!NOTE]
\> Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.

Reply:
Done.